modules/core/common/createReduxStore.ts

```diff
@@ -8,11 +8,63 @@ export const getStoreReducer = (reducers: any) =>
     ...reducers
   });
 
-const createReduxStore = (reducers: Reducer, initialState: DeepPartial<any>, routerMiddleware?: Middleware): Store => {
+const requestMiddleware: Middleware = _state => next => action => {
+  const { types, APICall, ...rest } = action;
+  if (!types) {
```

Comment:
Not sure I understood the role of the `types` object and why it's optional.

Reply:
_types_ is the optional property for asynchronous dispatches.
Synchronous dispatches should have only _type_ property